devito/symbolics/inspection.py

```diff
@@ -1,6 +1,5 @@
-from collections import Counter
-
 from sympy import cos, sin
+from collections import Counter
```

Comment:
there was a logic.

Counter is available in the standard Python lib; sympy is not.

I suggest to revert to what we had. This is how imports are organised in Devito

Reply:
Debug cos/sin reminders